tests/utils.js

```diff
@@ -6,9 +6,8 @@
 'use strict';
 
 import utils from '/ckeditor5/utils/utils.js';
-import utilsTestUtils from '/tests/utils/_utils/utils.js';
 
-const getIteratorCount = utilsTestUtils.getIteratorCount;
+const getIteratorCount = utils.count;
```

Comment:
Wouldn't it be better to use: `const count = utils.count;`? `getIteratorCount` was there because of the name of the method and now it's messy.